package.json

```diff
@@ -64,7 +65,7 @@
     "babel-plugin-transform-object-rest-spread": "^7.0.0-beta.3",
     "chai": "^4.1.0",
     "css-loader": "^0.28.7",
-    "electron": "2.0.2",
+    "electron": "^2.0.2",
```

Comment:
Can you undo this change? The `electorn-boilerplate` [recommends specifying an exact version here](https://github.com/szwacz/electron-boilerplate#upgrading-electron-version) and they point to the Electron docs.